packages/devtools/lib/src/ui/drag_scroll.dart

```diff
@@ -10,6 +10,11 @@ class DragScroll {
   /// Whether the element was dragged on the previous click.
   bool wasDragged = false;
 
+  /// Callback that will be called on vertical drag scroll.
```

Comment:
this comment is a bit low value.
Instead have a comment that explains the why.
from the
`VoidCallback onVerticalScroll;`
you can already tell this is a callback for a vertical scroll event.
The interesting part is why we need it instead of just letting the browser do its thing.
Would be also fine to just remove the comment.